bin/commands/push.sh

```diff
@@ -1,24 +1,33 @@
 #!/usr/bin/env bash
 
-source "$SCRIPTS_DIR/utils/utils.sh"
 source "$SCRIPTS_DIR/utils/docker.sh"
+source "$SCRIPTS_DIR/utils/environment.sh"
+source "$SCRIPTS_DIR/utils/image.sh"
 
 function push_cmd {
   local image_expr=$1
+  local push_options=${@:2}
 
+  local registry=$(get_registry)
+  local owner=$(get_owner)
+  local image_namespace=$(build_image_namespace $registry $owner)
+  
   IFS=' ' read -r image_name image_version <<< "$(parse_image_expr $image_expr)"
 
-  load_image_metadata $IMAGES_DIR $image_name
+  local image_paths=$(get_image_paths $IMAGES_DIR $image_name $image_version)
 
```

Comment:
Should have a `set -e` like [`bin/commands/build.sh:19`](https://github.com/cgerling/dockeryard/pull/13/files#diff-ed90beaaba91a9d91dd2ef2f0f694e03R19)